app/models/comment.rb

```diff
@@ -1,4 +1,5 @@
-class Comment < ApplicationRecord
+class Comment <
+  validates :text, presence: true
   belongs_to :user
   belongs_to :post
   after_save :update_comments_counter
```

Comment:
- [ ] After creating a post, I get this error when I try to access either the `Posts#Index` page or the `Posts#Show` pages. 

![image](https://github.com/NalianAchei2020/RubyonRails-blog/assets/77106962/9efe16f3-7cf1-41b7-b24e-05e297135674)
---

The `Comment` model class should inherit from the `ApplicationRecord` model class.